analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py

```diff
@@ -700,6 +707,11 @@ def construct_config_handler(cls, args):
                 isinstance(args.analyzer_config, list):
             for cfg in args.analyzer_config:
                 if cfg.analyzer == cls.ANALYZER_NAME:
+                    if cfg.option == 'executable':
```

Comment:
The probelm with this patch is that I can only specify the `gcc:executable` analyzer config if CodeChecker found already a gcc analyzer executable. Witch defies its purpos. Catch of 22...

The analyzer configs should be listable even if I don't have an analyzer binary. Analyzer plugins exist no matter if I have the analyzer on my manchine.

Please implement the following behaviour.

Let's assume I have no gcc in the PATH:
```
CodeChecker analyzers
clangsa 12.0.0 /usr/bin/clang
clang-tidy 11.0.0 /usr/bin/clang-tidy
cppcheck no analyzer binary found

CodeChecker --analyzer-config gcc

gcc:executable Use the specified analyzer binary. This supersedes any other method CodeChecker might use to get hold of one.
```